config/environments/development.rb

```diff
@@ -27,9 +27,19 @@
   end
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
 
-  config.action_mailer.perform_caching = false
+  config.action_mailer.perform_caching = true
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+     address: 'smtp.live.com',
+     port: 465,
```

Comment:
Does it work for you when you use 587?
I tried using the gmail smtp settings prior to the hotmail settings. and both 25 and 587 weren't working for me :(